translations/strings.yaml

```diff
@@ -277,6 +277,18 @@ notifications:
   unsecuredNetworkMessage:
     value: "%1 is not secure. Click here to turn on VPN and secure your device."
     comment: "%1 is the Wi-Fi network name"
+  VPNConnectedTitle: VPN Connected
+  VPNDisconnectedTitle: VPN Disconnected
+  VPNSwitchedServersTitle: VPN Switched Servers
+  connectedCity: 
+    value: "%2"
```

Comment:
Why are we exposing a string that only contains a variable? This should just be displayed directly from code.

The only exception is `%4, %5`, because locales need to be able to localize the comma and space. Still, variables should be %1 and %2.